ignore/src/types.rs

```diff
@@ -156,7 +156,7 @@ const DEFAULT_TYPES: &'static [(&'static str, &'static [&'static str])] = &[
     ("hs", &["*.hs", "*.lhs"]),
     ("html", &["*.htm", "*.html", "*.ejs"]),
     ("idris", &["*.idr", "*.lidr"]),
-    ("java", &["*.java", "*.jsp"]),
+    ("java", &["*.java", "*.jsp", "*.jspx", "*.properties", "pom.xml"]),
```

Comment:
I'm not well versed in Java, but [according to this page](https://maven.apache.org/guides/introduction/introduction-to-the-pom.html), `pom.xml` is an XML file. If I were a Java programmer, I'd be surprised if `rg -tjava` searched XML data. (Just like I'd be surprised if `rg -trust` searched `Cargo.toml`.)